fan-xiaomi.js

```diff
@@ -23,6 +27,20 @@ class FanXiaomi extends HTMLElement {
 
         const attrs = state.attributes;
 
+        if (attrs['model'] === 'dmaker.fan.1c'){
+            this.supportedAttributes.angle = false;
+            this.supportedAttributes.childLock = false;
+            this.supportedAttributes.rotationAngle = false;
+            this.supportedAttributes.speedLevels = 3;
+            this.supportedAttributes.natural_speed = false;
+            this.supportedAttributes.natural_speed_reporting = false;
+        }
+
+        if (attrs['model'] === 'dmaker.fan.p5'){
+            this.supportedAttributes.natural_speed_reporting = false;
```

Comment:
p5 fan supports natural_speed, but does not report raw speed. I changed to attributes natrual_speed, and natural_speed_reporting. for other fans, it's true true, for p5 it's true false, for 1c it's false false.